### PR TITLE
Issue #SB-26880 feat: Contextual help in sourcing and contribution flows

### PR DESCRIPTION
**Pull request by rajnishdargan**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-26880" title="SB-26880" target="_blank">StorySB-26880</a>  Contextual help in sourcing and contribution flows
  </td></table>
  <br />
 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions: Angular 8, Nodejs 12.16.1
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/client/src/app/modules/sourcing/components/chapter-list/chapter-list.component.ts [hunk 1]
```diff
@@ -125,6 +125,7 @@ export class ChapterListComponent implements OnInit, OnChanges, OnDestroy, After
   configUrl;
   tags = [];
   printUrl;
+  public helpSectionConfig = this.configService.appConfig.contextaulHelp.sourcing.reviewContributions;
```
Comment (vaivk369) [inline src/app/client/src/app/modules/sourcing/components/chapter-list/chapter-list.component.ts:128]: contextaulHelp  spelling is wrong @rajnishdargan

File: src/app/client/src/app/modules/shared/services/config/app.config.json [hunk 2]
```diff
@@ -1392,5 +1392,41 @@
     "gujarati": "gu",
     "punjabi": "pa",
     "oriya": "or"
+  },
+  "contextaulHelp": {
```
Comment (vaivk369) [inline src/app/client/src/app/modules/shared/services/config/app.config.json:1396]: contextaulHelp check spelling @rajnishdargan

File: src/app/client/src/app/modules/shared/services/config/app.config.json [hunk 3]
```diff
@@ -1392,5 +1392,41 @@
...
+        "helpMessage":  "You can invite users to your organization by clicking on Invite Users and sharing an invitation link with them"
+      },
+      "reviewContributions": {
+        "url": "https://dock.preprod.ntp.net.in/help/contribute/reviewer/review-contributions/index.html",
+        "helpHeader": "This table allow you to see a list of content pending for your review.",
+        "helpMessage":  "Click on any unit to proceed with the review process"
+      },
+      "assignUsersToProject": {
+        "url": "https://dock.preprod.ntp.net.in/help/contribute/how-to/how-to-assign-users-with-roles-to-contributor-projects.html",
+        "helpHeader": "This table allows you to see all the users available for this project and assign them relevant roles",
```
Comment (vaivk369) [inline src/app/client/src/app/modules/shared/services/config/app.config.json:1410]: these indexes can be , header, message and url

File: src/app/client/src/app/modules/sourcing/components/chapter-list/chapter-list.component.ts (hunk 1 above)
Comment (rajnishdargan) [reply src/app/client/src/app/modules/sourcing/components/chapter-list/chapter-list.component.ts]: done

File: src/app/client/src/app/modules/shared/services/config/app.config.json (hunk 3 above)
Comment (rajnishdargan) [reply src/app/client/src/app/modules/shared/services/config/app.config.json]: done

File: src/app/client/src/app/modules/contribute/components/org-user-list/org-user-list.component.ts [hunk 4]
```diff
@@ -40,6 +40,7 @@ export class OrgUserListComponent implements OnInit, AfterViewInit {
   isInitialSourcingOrgUser = true;
   telemetryPageId: string;
   public impressionEventTriggered: Boolean =  false;
+  public helpSectionConfig = this.configService.appConfig.contextualHelp.contribute.manageUsers;
```
Comment (pallakartheekreddy) [inline src/app/client/src/app/modules/contribute/components/org-user-list/org-user-list.component.ts:43]: @rajnishdargan this.configService.appConfig?.contextualHelp? contribute or use loads function _.get

**Review by pallakartheekreddy — APPROVED** [review]

**Pull request merged**